### PR TITLE
[RFR] Read analysis data from analysis.json

### PR DESCRIPTION
**Pull request by midays**
Signed-off by midays@redhat.com

### CODE REVIEW

**Comment by midays** [comment]
- Dynamically create test runs
- update config file name
- add open_report_page

File: src/tests/test_analysis_intellij.py [hunk 1]
```diff
@@ -1,64 +1,32 @@
...
-        project=config_data["project_paths"]["eap7_generic"],
-        migration_target="eap7",
-    )
-    assert intellij.is_analysis_complete()
-    assert intellij.verify_story_points(target="eap7")
 
-
-def test_analysis_eapxp(setup_intellij):
+@pytest.mark.parametrize("app_name", json.load(open(DATA_DIR + "analysis.json")))
+def test_analysis_eap(setup_intellij, app_name, analysis_data):
```
Comment (sshveta) [inline src/tests/test_analysis_intellij.py]: We can change the name of this test to 
 "test_different_analysis"  As it is running analysis on various targets and sources .

File: src/tests/test_analysis_intellij.py [hunk 2]
```diff
@@ -1,64 +1,32 @@
...
     """
-    Test to run analysis on ruleset thorntail to eapxp2 in IntelliJ IDE
+    Analysis tests for VScode using various migration paths
```
Comment (sshveta) [inline src/tests/test_analysis_intellij.py]: This is for intelliJ right??

**Review by sshveta — CHANGES_REQUESTED** [review]
Looks good @midays Few changes required.

File: src/tests/test_analysis_intellij.py (hunk 2 above)
Comment (midays) [reply src/tests/test_analysis_intellij.py]: done

File: src/tests/test_analysis_intellij.py (hunk 1 above)
Comment (midays) [reply src/tests/test_analysis_intellij.py]: done, changed naming to **test_analysis_intellij** to match up with **test_analysis_vscode**

**Review by nachandr — APPROVED** [review]
LGTM

**Review by sshveta — APPROVED** [review]

**Pull request merged**